lms/djangoapps/courseware/module_render.py

```diff
@@ -748,10 +748,12 @@ def xqueue_callback(request, course_id, userid, mod_id, dispatch):
     #               'xqueue_body'  : 'Message from grader'}
     for key in ['xqueue_header', 'xqueue_body']:
         if key not in data:
+            log.error(u"xqueue_callback missing {}".format(key))
```

Comment:
these are going to fail the quality build because you're not using the lazy logging format.

https://build.testeng.edx.org/job/edx-platform-all-tests-pr/1621/SHARD=1,TEST_SUITE=quality,label_exp=jenkins-worker/artifact/reports/diff_quality/diff_quality_pylint.html

L734 won't fail because you do the interpolation in the line before, but if `msg` in L733 isn't used elsewhere I recommend moving it into the logging call with the %s formatting.